frontend/src/scenes/dashboard/DashboardItem.js

```diff
@@ -149,6 +155,7 @@ export function DashboardItem({
                                         router.actions.push(link)
                                     }
                                 }}
+                                style={{ fontSize: 16, fontWeight: 'bold' }}
```

Comment:
Bold (700) seems to heavy to me, I'd go for 500 like in the top bar (project switcher et al.)

Reply:
Not entirely sure, but anyways once we introduce standardized components, it may change so updating for now.